app/controllers/Authentication.scala

```diff
@@ -254,6 +255,31 @@ class Authentication @Inject() (
     )
   }
 
+  def getToken = SecuredAction.async { implicit request =>
+    for{
+      maybeOldAuthenticator <- env.combinedAuthenticatorService.tokenDao.findByLoginInfo(request.identity.loginInfo)
```

Comment:
what is an authenticator? here the word seems synonymous with token?

Reply:
An authenticator can be a token or a cookie. But because we are asking the tokenDAO if there is a token, we already know that we are getting a token (and not a cookie) (or it is None)
So you are right 'maybeOldToken' would be a better name.